addons/survey/static/src/js/survey_quick_access.js

```diff
@@ -8,9 +8,10 @@ publicWidget.registry.SurveyQuickAccessWidget = publicWidget.Widget.extend({
     events: {
         'click button[type="submit"]': '_onSubmit',
         'input #session_code': '_onSessionCodeInput',
+        'click .o_survey_session_launch_session': '_launch_session',
```

Comment:
2 things here:

- Click handler are usually named "onXXXClick" (convention).
- Within javascript code, we use camelCase and not snake_case